hw3/scripts/HW3_Code_TM.txt

```diff
@@ -0,0 +1,47 @@
+cd HW3/betas
+
+3dmask_tool -input ../masks/full_mask.sub-*.HEAD -frac 1.0 -prefix mask_group 
+
+3dinfo -label stats.sub-08_REML+tlrc.BRIK
+
+3dinfo -label2index 'Words#0_Coef' stats.sub-08_REML+tlrc.BRIK
+
+3dttest++ -mask mask_group+tlrc -prefix words-fixation -setA 'stats*_REML+tlrc.HEAD[Words#0_Coef]'
+
+for f in ../blurs/blur_est.sub-*.1D; do 1d_tool.py -infile $f -select_row 3 -write >> blurs.acf.1D done 
```

Comment:
The effect here is minor, but row index 3 contains the blur estimates from OLS. I have given you the REML results, so you need to choose a different row from the file.